src/Chapter3.hs

```diff
@@ -480,6 +524,46 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city totally.
 -}
 
+data Inhabitants = One | Two | Three | Four deriving (Show)
+inhabitantsToInt :: Inhabitants -> Int
```

Comment:
Better solution possible after knowledge from of Task 8

Reply:
Yes, indeed, you can derive `Enum` which gives you such function almost for free. But it's good to practice plain pattern matching first :slightly_smiling_face: